packages/fetchai/connections/p2p_libp2p/dht/dhtnode/message.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package dhtnode;
+
+message AgentRecord {
+    string service_id = 1;
+    string address = 2;
+    string public_key = 3;
+    string peer_public_key = 4;
+    string signature = 5;
+}
+
+message Register {
+    AgentRecord record = 1;
+}
+
+message LookupRequest {
+    string agent_address = 1;
+}
+
+message LookupResponse {
+    AgentRecord agent_record = 1;
+}
+
+message AeaEnvelope {
+    // TOFIX(LR) import aea.Envelop type
+    bytes envel = 1;
```

Comment:
need to find a way to import this proto file from `message.proto` one to avoid double serialization of `aea.Envelope`